koku/api/report/queries.py

```diff
@@ -389,8 +426,12 @@ def _set_or_filters(self, or_filter=None):
 
         return filters.compose(logical_operator="or")
 
-    def _set_tag_exclusion_filters(self):
-        """Creates exclusion fitlers for tags that allow null returns.
+    def _build_prefixed_based_exclusions(self, db_column, exclude_filters):
```

Comment:
There is little consistency between all these function names. If we update these
```
_build_prefixed_based_exclusions -> _set_prefix_based_exclusions
_set_operator_specific_prefixed_filters -> _set_operator_specific_prefix_based_filters
```
it will bring things in line with `_set_prefix_based_filters`.